src/xpring-client.ts

```diff
@@ -18,11 +19,17 @@ class XpringClient {
    * The XpringClient will use gRPC to communicate with the given endpoint.
    *
    * @param grpcURL The URL of the gRPC instance to connect to.
+   * @param useNewProtocolBuffers If `true`, then the new protocol buffer implementation from rippled will be used. Defaults to false.
    */
-  public constructor(grpcURL: string) {
-    const defaultXpringClient = LegacyDefaultXpringClient.defaultXpringClientWithEndpoint(
+  public constructor(grpcURL: string, useNewProtocolBuffers: boolean = false) {
+    var defaultXpringClient: XpringClientDecorator = LegacyDefaultXpringClient.defaultXpringClientWithEndpoint(
```

Comment:
I believe let would be more appropriate here.

Reply:
Prettier actually re-wrote this when I merged in #139. Sorry about that.